test/bin/ci_phase_iso_boot.sh

```diff
@@ -9,6 +9,8 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=test/bin/common.sh
 source "${SCRIPTDIR}/common.sh"
 
+ENABLE_MIRROR=${ENABLE_MIRROR:-true}
```

Comment:
Do you think we should be more specific on what mirror we're enabling?
Like, ENABLE_MIRROR_IMAGES or ENABLE_REGISTRY_MIRROR

Reply:
Done. Also added some documentation to readme.